tests/test_files/invalid.malformed.parser.error.yaml

```diff
@@ -0,0 +1,54 @@
+- apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Could you add a comment highlighting where the yaml is malformed? This will help debugging issues in the future.